.github/workflows/rollbackDB.yml

```diff
@@ -56,7 +56,7 @@ jobs:
           creds: ${{ secrets.AZURE_CREDENTIALS }}
       - uses: hashicorp/setup-terraform@v1
         with:
-          terraform_version: 0.15.1
+          terraform_version: 1.1.4
```

Comment:
wow thats quiet the jump! does this mean the current job rollbackDB is broken due to mismatched TF version, does it make sense to merge this by itself?

Reply:
It does, and actually yes, it does since I didn't think this PR would take this long. I'll submit a separate PR for this.